src/api/.rubocop.yml

```diff
@@ -81,3 +81,254 @@ Rails/BulkChangeTable:
     - 'db/migrate/20170912140257_change_kiwi_packages_columns_from_big_int_to_int.rb'
     - 'db/migrate/20171107125828_rename_kiwi_preference_types_to_kiwi_preferences.rb'
     - 'db/migrate/20180516074538_explicitly_set_charset.rb'
+
+Style/MethodCallWithArgsParentheses:
+  Enabled: true
+  Exclude:
+    - 'app/views/**/*'
+    - 'db/migrate/**/*'
+  # the cop doesn't come with a well defined list of good methods, so this
+  # list is the list of offending functions when introducing. Removing functions
+  # from here leads to autocorrecting calls, if you call a new DSL function, add
+  # it here
+  IgnoredMethods:
+    -  abort
+    -  acceptinfo
+    -  acronym
+    -  action
+    -  add_attribute
+    -  add_column
+    -  add_element
+    -  add_foreign_key
+    -  add_index
+    -  add_reference
+    -  addreview
+    -  add_user
+    -  add_watched_project
+    -  after_action
+    -  and_return
+    -  archfilter
+    -  around
+    -  assert
+    -  assert_equal
+    -  assert_kind_of
+    -  assert_match
+    -  assert_nil
+    -  assert_not
+    -  assert_not_equal
+    -  assert_not_nil
+    -  assert_no_xml_tag
+    -  assert_project
+    -  assert_redirected_to
+    -  assert_response
+    -  assert_select
+    -  assert_xml_tag
+    -  author
+    -  authorize
+    -  basic_auth
+    -  belongs_to
+    -  boolean
+    -  build
+    -  call
+    -  category
+    -  change_column
+    -  change_column_default
+    -  change_column_null
+    -  clean_older_than
+    -  click_button
+    -  click_link
+    -  column
+    -  command_name
+    -  comment!
+    -  commit
+    -  concat
```

Comment:
I would drop `concat` here. It's just one of many Array methods.

Not sure where `comment!` and `commit` are coming from, but if that's OBS code I would drop that as well.

Reply:
But it's only 2 usages, so I removed it too